package.json

```diff
@@ -153,68 +153,74 @@
-    "@storybook/addon-viewport": "7.4.6",
-    "@storybook/angular": "7.4.6",
-    "@storybook/api": "7.4.6",
-    "@storybook/core-server": "7.4.6",
-    "@storybook/preview-api": "7.4.6",
-    "@storybook/types": "7.4.6",
-    "@trivago/prettier-plugin-sort-imports": "4.2.1",
-    "@types/cross-spawn": "6.0.3",
+    "@storybook/addon-a11y": "^7.6.10",
+    "@storybook/addon-actions": "^7.6.10",
+    "@storybook/addon-controls": "^7.6.10",
+    "@storybook/addon-toolbars": "^7.6.10",
+    "@storybook/addon-viewport": "^7.6.10",
+    "@storybook/angular": "^7.6.10",
+    "@storybook/api": "^7.6.10",
+    "@storybook/core-server": "^7.6.10",
+    "@storybook/preview-api": "^7.6.10",
+    "@storybook/types": "^7.6.10",
+    "@trivago/prettier-plugin-sort-imports": "4.3.0",
+    "@types/cross-spawn": "6.0.6",
     "@types/dompurify": "3.0.3",
     "@types/dragula": "2.1.36",
     "@types/fontfaceobserver": "2.1.1",
-    "@types/fs-extra": "11.0.2",
+    "@types/fs-extra": "11.0.4",
     "@types/glob": "8.1.0",
     "@types/google-libphonenumber": "7.4.28",
     "@types/he": "1.2.1",
-    "@types/jasmine": "5.1.0",
-    "@types/jest": "29.5.5",
+    "@types/jasmine": "5.1.4",
+    "@types/jest": "29.5.11",
     "@types/node": "18.17.8",
     "@types/validator": "13.11.2",
     "@typescript-eslint/eslint-plugin": "6.19.0",
     "@typescript-eslint/parser": "6.19.0",
     "cross-spawn": "7.0.3",
     "cypress": "13.6.3",
-    "eslint": "8.51.0",
+    "eslint": "8.56.0",
     "eslint-config-prettier": "9.1.0",
-    "eslint-plugin-cypress": "2.12.1",
-    "eslint-plugin-deprecation": "1.4.1",
+    "eslint-plugin-cypress": "2.15.1",
+    "eslint-plugin-deprecation": "2.0.0",
     "eslint-plugin-storybook": "0.6.15",
-    "fs-extra": "11.1.1",
+    "fs-extra": "11.2.0",
     "glob": "10.3.10",
     "jasmine": "5.1.0",
     "jasmine-core": "5.1.1",
     "jest": "29.7.0",
     "jest-environment-jsdom": "29.7.0",
     "jest-environment-node": "29.7.0",
-    "jest-preset-angular": "13.1.6",
+    "jest-preset-angular": "14.0.0",
     "karma": "6.4.2",
     "karma-chrome-launcher": "3.2.0",
     "karma-coverage": "2.2.1",
     "karma-jasmine": "5.1.0",
     "karma-jasmine-html-reporter": "2.1.0",
-    "ng-packagr": "16.2.3",
+    "ng-packagr": "17.1.1",
     "nx": "17.2.8",
     "nyc": "15.1.0",
-    "postcss": "8.4.31",
-    "postcss-import": "14.1.0",
-    "postcss-preset-env": "7.8.3",
+    "postcss": "8.4.33",
+    "postcss-import": "16.0.0",
+    "postcss-preset-env": "9.3.0",
     "postcss-url": "10.1.3",
     "prettier": "2.8.8",
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "resize-observer-polyfill": "1.5.1",
-    "storybook": "7.4.6",
-    "storybook-addon-angular-router": "1.9.2",
-    "ts-jest": "29.1.1",
-    "ts-node": "10.9.1",
-    "typescript": "5.1.6"
+    "storybook": "^7.6.10",
+    "ts-jest": "29.1.2",
+    "ts-node": "10.9.2",
+    "typescript": "5.3.3"
+  },
+  "overrides": {
+    "ng2-dragula@5.0.1": {
```

Comment:
What's the reason for this change?

Reply:
`ng2-dragula` does not yet have Angular 17 support. The [Angular 17 support PR](https://github.com/valor-software/ng2-dragula/pull/1101) is currently in progress. This is a workaround.